libyul/backends/evm/OptimizedEVMCodeTransform.cpp

```diff
@@ -192,14 +212,9 @@ void OptimizedEVMCodeTransform::assertLayoutCompatibility(Stack const& _currentS
 AbstractAssembly::LabelID OptimizedEVMCodeTransform::getFunctionLabel(Scope::Function const& _function)
 {
 	CFG::FunctionInfo const& functionInfo = m_dfg.functionInfo.at(&_function);
-	if (!m_functionLabels.count(&functionInfo))
-		m_functionLabels[&functionInfo] = m_useNamedLabelsForFunctions ? m_assembly.namedLabel(
-			functionInfo.function.name.str(),
-			functionInfo.function.arguments.size(),
-			functionInfo.function.returns.size(),
-			{}
-		) : m_assembly.newLabelId();
-	return m_functionLabels[&functionInfo];
+	if (AbstractAssembly::LabelID const* label = util::valueOrNullptr(m_functionLabels, &functionInfo))
+		return *label;
+	yulAssert(false, "");
```

Comment:
Why not just use `.at(...)`?

Reply:
Because that asserts in the STL, while this asserts where we know what happens...
Fine with using ``.at``, though, but are we fine with relying on STL exceptions and assertions these days in general :-)?

Reply:
But well, it shouldn't ever actually fail anyways, so it doesn't matter.

Reply:
The only difference is that the exception is not as nicely formatted, but most of the time, the exception information does not help users anyway and we can easily hook it into a debugger. I would say: If it is more readable, prefer stl exceptions.